server.js

```diff
@@ -5590,6 +5590,57 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Swagger Validator integration.
+camp.route(/^\/swagger\/(v)\/(2\.0)\/(https?)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var type = match[1];        // e.g. `v` for validate
+  var specVer = match[2];     // e.g. `2.0` for OpenAPI 2.0
+  var scheme = match[3];      // e.g. `https`
+  var swaggerUrl = match[4];  // e.g. `api.example.com/swagger.yaml`
+  var format = match[5];
+
+  var badgeData = getBadgeData('swagger', data);
+
+  var urlParam = encodeURIComponent(scheme + '://' + swaggerUrl);
+  var url = 'http://online.swagger.io/validator/debug?url=' + urlParam;
```

Comment:
Indeed, that threw me off when I was initially troubleshooting the issue.  I didn't dig into whether browsers are shipping some intermediate certificates or if they have extra logic for fetching them dynamically.  It seems to be consistent across all of the browsers I tested.  Now I'm curious again...